editor/src/components/editor/notification-bar.tsx

```diff
@@ -43,16 +40,9 @@ export const LoginStatusBar = React.memo(() => {
     case 'LOGGED_IN':
       return null
     case 'OFFLINE_STATE':
-      return <EditorOfflineBar />
+      return null
     case 'NOT_LOGGED_IN':
-      return (
-        <NotificationBar
-          level='PRIMARY'
-          message={'Welcome to Utopia. Click here to sign in and save your projects.'}
-          onClick={onClickLoginNewTab}
-          style={{ cursor: 'pointer' }}
-        />
-      )
+      return null
```

Comment:
We still have other notification bars - do we still want to keep those for the moment?

Reply:
Yes, for now. Will likely be replaced by future title bar updates